Tests/Functional/Domain/Repository/Product/TeaRepositoryTest.php

```diff
@@ -66,6 +66,15 @@ public function findAllSortsByTitleInAscendingOrder(): void
         self::assertSame(2, $result->current()->getUid());
     }
 
+    /**
+     * @test
+     */
+    public function findByUidForNoneExistingRecord(): void
+    {
+        $model = $this->subject->findByUid(1);
```

Comment:
Please let's add a blank line to visually separate the "execute" step from the "verify" step.

https://speakerdeck.com/oliverklee/test-driven-development-with-phpunit-915a5f2c-3d37-4e41-b5bc-ef1efca9c01e?slide=22